internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -50,16 +51,15 @@ func HelpProvider(epa *externalplugins.ConfigAgent) func(
 			}
 		}
 		pluginHelp := &pluginhelp.PluginHelp{
-			Description: "The ti-community-lgtm plugin manages the application and " +
-				"removal of the 'status/LGT{number}' (Looks Good To Me) label which is typically used to gate merging.",
-			Config: configInfo,
+			Description: "The ti-community-lgtm plugin manages the 'status/LGT{number}' (Looks Good To Me) label.",
+			Config:      configInfo,
 		}
 
 		pluginHelp.AddCommand(pluginhelp.Command{
-			Usage:       "/lgtm [cancel] or GitHub Review action",
-			Description: "Adds or removes the 'status/LGT{number}' label which is typically used to gate merging.",
+			Usage:       "/lgtm [cancel] or triggers by github review action.",
```

Comment:
```suggestion
			Usage:       "/lgtm [cancel] or triggers by GitHub Review action.",
```

Reply:
I don't think it's a proper noun. Can you find any github documentation that contains `Github Review`? k8s is not always correct.

Reply:
We use that description in all our documentation.

https://book.prow.tidb.io/plugins/lgtm.html

Reply:
And `Github` or `github` should be `GitHub`.
<img width="246" alt="image" src="https://user-images.githubusercontent.com/29879298/101868228-d4318080-3bb7-11eb-93b1-b79711081bab.png">

Reply:
If you feel this description is incorrect, please find an appropriate description and refine all documentation.

Reply:
done

Reply:
Thanks!